python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -60,7 +60,7 @@ def get(self, mine_no_or_guid=None):
             # Handle MapView request
             _map = request.args.get('map', None, type=str)
             if _map and _map.lower() == 'true':
-                records = MineMapViewLocation.query.all()
+                records = MineMapViewLocation.query.filter(MineMapViewLocation.latitude != None)
```

Comment:
Can you modify the view's query to include a where lat and long are not null clause instead?

Reply:
db view will need to be updated when mine_location and mine tables are merged. (and possibly obsolete?). This is the quickest fix for the current issue for end of sprint. I don't disagree though.